### PR TITLE
Version 0.3.5

### PR DESCRIPTION
**Pull request by jheek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,31 +5,52 @@ vNext
...
+- 
+- 
+
+
+
+0.3.5
+------
+
+
  - `flax.linen.Conv` no longer interprets an int past as kernel_size as a 1d convolution. Instead a type error is raised stating that
```
Comment (andsteing) [inline CHANGELOG.md:39]: a bit hard to parse ... maybe it should be "an int passed as" ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,31 +5,52 @@ vNext
...
+- 
+- 
+- 
+- 
+- 
+- 
+
+
+
+0.3.5
```
Comment (andsteing) [inline CHANGELOG.md:35]: Could you group these for easier reading?
(maybe "breaking changes", "new features", and "bugfixes")

**Review by andsteing — APPROVED** [review]
Thanks for starting the release!

Some minor comments...

File: CHANGELOG.md (hunk 2 above)
Comment (jheek) [reply CHANGELOG.md:35]: Done
